plugins/caretposition/plugin.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+'use strict';
+
+( function() {
+
+	CKEDITOR.plugins.add( 'caretposition', {} );
+
+	var isMSSelection = typeof window.getSelection != 'function';
+
+	/**
+	 * Returns the position of the caret.
+	 *
+	 * **Note:** In IE8 this method always returns the position of the real selection,
+	 * while in other browsers it will stick to the position of the range returned
+	 * by {@link #getRanges}, which may be one for a {@link #lock locked} or a {@link #fake fake} selection.
+	 *
+	 * **Note:** See {@link CKEDITOR.dom.range#getClientPosition} for information about possible issues
+	 * that may be caused by serious browser bugs.
+	 *
+	 * @returns {Object} The caret position
```

Comment:
Again, missing `@since` tag. Please review the entire docs.

Reply:
`caretposition` will be probably removed so I will skip fixing it and its dependencies (e.g. tests).